backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/utils/StoreFieldCleaner.kt

```diff
@@ -0,0 +1,14 @@
+package app.ehrenamtskarte.backend.stores.utils
+
+/** Returns null if string can't be trimmed f.e. empty string
+ * Removes subsequent whitespaces
+ * */
+fun String?.clean(removeSubsequentWhitespaces: Boolean = true): String? {
```

Comment:
nit-picky:
if we call this function with `removeSubsequentWhitespaces = false`, we get the same effect as trim().
so I'm not quite sure why this extra branch of logic is needed?
when we need trim() why not just call trim() ?

Reply:
i adjusted this function to return null for empty string and added removeSubsequentStrings `false` if it is used for the csv import via administration

Reply:
can you give me an explanation about what that `notes` should be that will be replaced with this function. i couldn't find any good example for that.

Reply:
i readded `replaceNa()` just in case to not break anything.
It would be good to add an example in the tests @steffenkleinle. But this can be done is a separate task
I couldn't find much information about that and how a string with such a note looks like